emloop_tensorflow/model.py

```diff
@@ -223,12 +226,25 @@ def run(self, batch: el.Batch, train: bool=False, stream: el.datasets.StreamWrap
                 for output_name in self.output_names:
                     fetches.append(tower[output_name])
 
+        run_options = None
```

Comment:
This is more or less the same code as in `FrozenModel`. Can you perhaps wrap `session.run` to some util function which would be utilized in both classes?